libdnf-cli/output/repoquery.hpp

```diff
@@ -46,8 +46,10 @@ static struct libscols_table * create_package_info_table(Package & package) {
     add_line_into_package_info_table(table, "Version", package.get_version().c_str());
     add_line_into_package_info_table(table, "Release", package.get_release().c_str());
     add_line_into_package_info_table(table, "Architecture", package.get_arch().c_str());
-    auto size = package.get_size();
-    add_line_into_package_info_table(table, "Size", std::to_string(size).c_str());
+    add_line_into_package_info_table(table, "Install size", std::to_string(package.get_install_size()).c_str());
+    if (!package.is_installed()) {
+        add_line_into_package_info_table(table, "Package size", std::to_string(package.get_package_size()).c_str());
+    }
```

Comment:
Not sure what this method is for, I haven't found it used. I was just thinking adding the column conditionally based on whether the package is installed is non-deterministic for the user (in the way they won't know whether the column will or will not be in the output). Also not sure how this behaves when there are some packages that are installed and some that aren't?

Reply:
If a package is available, then the "Package size" is printed.
If it is installed, then we don't have the package, therefore we can't print anything.
In my opinion it's still better than simply printing Size: <install or package size> - which is totally ambiguous.